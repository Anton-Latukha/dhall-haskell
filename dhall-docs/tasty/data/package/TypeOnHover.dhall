{-|
This example shows the type-on-hover feature over builtins
-}
let true = True
let false = False

in 1 + 3
