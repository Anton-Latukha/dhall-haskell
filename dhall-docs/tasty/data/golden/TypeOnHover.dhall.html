<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/TypeOnHover.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="https://unpkg.com/@popperjs/core@2"
              type="text/javascript"
    /><script src="https://unpkg.com/tippy.js@6" type="text/javascript"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >TypeOnHover.dhall</span></h2
      ><a data-path="/TypeOnHover.dhall" class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
        ><p
        >This example shows the type-on-hover feature over builtins</p>
</div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          >{-|<br
          />This example shows the type-on-hover feature over builtins<br
          />-}<br
          />let true = <span data-tippy-content="&lt;pre&gt;Bool&lt;br&gt;&lt;/pre&gt;"
                             class="type-tooltip"
          >True</span
          ><br
          />let false = <span data-tippy-content="&lt;pre&gt;Bool&lt;br&gt;&lt;/pre&gt;"
                              class="type-tooltip"
          >False</span
          ><br
          /><br
          />in <span data-tippy-content="&lt;pre&gt;Natural&lt;br&gt;&lt;/pre&gt;"
                     class="type-tooltip"
          >1</span
          > + <span data-tippy-content="&lt;pre&gt;Natural&lt;br&gt;&lt;/pre&gt;"
                    class="type-tooltip"
          >3</span
          ><br/></pre></div></div></body></html>